elastic/beats
# Seperate namespaces for V1 and V2 controller paths

## What does this PR do?
I'm working on some new automation to test cgroups code on different platforms + containers, and this was the first bug I noticed. On hybrid systems, the code that fetches the Controller paths can overwrite the `cpu` controller on V1 if it exists on V2, as it shares a name with the V1 controller. That's bad.

## Checklist


## How to test this PR locally

- build and run on a linux system running hybrid cgroups (for example, Ubuntu 18.04)
- Run `system/process`, ensure that the `cpu` fields in `cgroup` is populated with valid data
